js/iframe-coordinator/libs/client.ts

```diff
@@ -23,10 +23,45 @@ function start(expectedOrigin) {
         msg: location
       });
     });
+
+    /**
+     * Request a toast message be displayed by the host.
+     * 
+     * The page embedding the host is responsible for handling the fired custom event and
+     * presenting/styling the toast.  Application-specific concerns such as level, TTLs,
+     * ids for action callbacks (toast click, toast action buttons), etc. can be passed via custom
+     * properties of the options param.
+     * 
+     * @param {string} message - The message content of the toast
+     * @param {object=} options - Supplimental toast options.
+     * @param {string=} options.title - Optional title for the toast.
+     * @param {*=} options.x - Optional, application-specific toast properties.  Note: Must be JSON serializable.
```

Comment:
Did a fair amount of looking, but I'm not sure dynamic properties are supported in jsdoc.  Let me know if y'all know of a supported syntax.  If not, this should be pretty obvious from a textual perspective what we're after.

Reply:
Hopefully we can specify it nicely in typescript and tsdoc can make it clear for us.